### ISSUE TITLE
Error in line 54

### ISSUE THREAD

**Issue by gru-github** [AndyTaylorTweet/Pi-Star_Binaries_sbin#59]
The shell variable
  **_ipvar_**
should be protected with double quotes.
In my case this variable was empty and the comparison was syntactically wrong and the script failed.

**Comment by MW0MWZ** [comment]
please review the commit above.

**Comment by gru-github** [comment]
Okay so far, but I've seen two other potential problems.
The assigned initial value of 
  _**ipVar**_
in line 24 is different from the assignment in line 56. I don't see why.
And the comparison in line 54 with " " (one space) is wrong. An empty string is returned by
 _**hostname -I**_
if no address is assigned.

Sorry, should have seen it earlier...

**Comment by MW0MWZ** [comment]
line 24 of what file? this is used a few times, can you give me the file so that I can follow along too.

**Comment by gru-github** [comment]
mmdvmhost.service

**Comment by MW0MWZ** [comment]
${ipVar} is the same as $ipVar, nothing wrong with that bit.
Line 56 isn't a test, it's re-setting the variable, so those two are OK.

I'll double check the output for hostname with and without network.

**Comment by gru-github** [comment]
I checked it, In my linux/pi-star version 
`ifconfig wlan0 down`
`echo -n $(hostname -I)|hd`
returns nothing.
`vvv=$(hostname -I)`
echo ${#vvv}
returns zero

**Comment by gru-github** [comment]
I checked on my pi-star version the  
* consequences of "" in the until expression
* and the removal of the cut command
It works for both cases, local AP mode and Wifi connected to AP.

I haven't checked the timeout value of the mmdvmhost systemd service config which should be way higher than the automatic AP-mode switch timeout.

There might be now conflict potential here because the until loop was always left.

**Comment by MW0MWZ** [comment]
This is a solution to an OLD problem, and the concept of "wait for an IP" might not matter so much any more.
I might not make changes today, but I'll look at what can be done with this. MMDVMHost should not care for example if the local AP is running or not, and maybe a better solution is to have systemd wait until the network target is reached before starting the radio related services. if there is no working network - it should not matter if MMDVMHost is running or not.

**Comment by gru-github** [comment]
You're right, it's not urgent and putting everything in systemd's hands is a good idea. Very flexible that piece.

Nevertheless I tested another non-standard case with the changes above made to mmdvmhost.service script (cut removed, empty string comparison):
  * Extern AP is off
  * Booting Pi-star
  * waiting for Pi-star AP coming up
   -> two ip addresses returned by hostname   -I
  -> that was probably the reason for the cut in the old days
  * Pi-star AP (50.1) is up and running
  * now switching on the external AP
  * after a while the external AP is connected, but the local AP (50.1) is still there.
  * Display is not updated accordingly 
  * now it gets erratic. External AP connection is coming constantly on and off and on....
 
Another reason to change the systemd config...
